types/src/grants.d.ts

```diff
@@ -12,3 +12,20 @@ export type GrantObject = {
 export type GrantArray = [BigNumber, string, string, string];
 export type GrantEthers = GrantArray & GrantObject;
 export type Grant = GrantObject | GrantEthers;
+export type GrantRound = {
+  address: string,
+  owner: string,
+  registry: string,
+  donationToken: string,
+  donationTokenName: string,
+  donationTokenSymbol: string,
+  donationTokenDecimals: BigNumberish,
```

Comment:
Same comment as earlier on using `TokenInfo` type

Reply:
Resolved in b35f417